src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -110,6 +150,15 @@ def sessions_errored_set(org_id: int, metric_ids, alias=None):
     )
 
 
+def all_transactions(org_id, metric_ids, alias=None):
+    return _dist_count_aggregation_on_tx_status_factory(
+        org_id,
+        exclude_tx_statuses=[],
```

Comment:
I am not a fan of adding empty list in a `notIn` condition.. I am actually surprised clickhouse didn't error .. usually it does if you send in an empty list in the condition